Slim/Menu/BrowseLibrary.pm

```diff
@@ -1795,6 +1795,8 @@ sub _tracks {
 					delete $_->{'artwork_track_id'};
 					$_->{'playall'} = 1;
 				}
+
+				$_->{'favorites_url'} = $_->{'url'};
```

Comment:
Why not just use the `item.url` in the template? Would save one more line change.

Reply:
You're right. I added that before full investigation, just in case its existence solved the problem. I'll remove it.